econet_hvac.yaml

```diff
@@ -13,16 +13,6 @@ dashboard_import:
 
 econet:
   dst_address: 0x380
-  on_datapoint_update:
```

Comment:
This removes the only usage of this. In the absence of documentation it might be good to keep it commented out to have an example how to parse RAW. Maybe comment out (or remove) the 2 lines below that call publish state and keep the logging?